docker-compose.env.sample

```diff
@@ -1 +1,5 @@
-DATABASE_URL=postgres://postgres@db:5432/DataSubmissionServiceApi_development?template=template0&pool=5&encoding=unicode
+DATABASE_URL=
```

Comment:
Could you restore the example `DATABASE_URL`? - it's for development only, and there's no harm it being here.